custom_components/dropboxbackup/backup.py

```
@@ -14,7 +14,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 # Dropbox recommends 4 MB chunks for upload sessions.
```

Ruff (0.14.10)

Comment contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF003)